tests/test_hear_data_processing.py

```
@@ -55,16 +58,18 @@ def test_multicast_sender_with_no_data(self) -> None:
 		theResult = False
 		fail_fixture = "SAY -X] RECV? != error"
 		_fixture_port_num = self._always_generate_random_port_WHEN_called()
+		_fixture_mcast_addr = "224.0.0.1"
 		try:
 			self.assertIsNotNone(_fixture_port_num)
 			self.assertIsInstance(_fixture_port_num, int)
+			self.assertIsNotNone(_fixture_mcast_addr)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)